src/test_utils/es/es_test_cluster.js

```diff
@@ -0,0 +1,87 @@
+import { resolve } from 'path';
+import { esTestServerUrlParts } from '../../../test/es_test_server_url_parts';
+
+import libesvm from 'libesvm';
+
+const ESVM_DIR = resolve(__dirname, '../../../esvm/test_utils/es_test_cluster');
+
+export class EsTestCluster {
+  _freshBranches = [];
+  _directory = resolve(ESVM_DIR);
```

Comment:
Use to be making modifications based on test cluster `name`, but refactored so that all test clusters share a directory. will remove